### PR TITLE
fix for navBar style in mobile devices

### LINKED ISSUE #213: Fix the navbar
### Design of New Navbar:
https://www.figma.com/file/bpFHG9hXM13GSLmTkoNsrF/Navbar?node-id=0%3A1

## Few Details 
- font : Roboto
- weight: 700
- active color : #49A82E
- hover color : #49A82E
- font size : 16 px (try making it in rem)
- active underline height : 3px (color : #49A82E)
- RDS Logo Size - 50x50

### Pay Attention
=> Underline should be under the text as shown in the image and not inside the container in which the text persists.
=> Underline should be only under the active tab, it should not be visible on hover
=> Clicking on RDS logo we should be redirected to https://www.realdevsquad.com/
=> The full navbar should be changed to the toggle menu when the screen width is less than 970px.

### PR DESCRIPTION
**Pull request by jjagannn**
closes #213 
iPhone X view before changes:
image

iPhone X view after changes:
image

Samsung Galaxy fold view before changes:
image

Samsung Galaxy fold view after changes:
image

Note:
The rest of the devices were aligned as expected.

Fix for all screens to vertically centre the text.
Before changes:
image

After changes:
image

### CODE REVIEW

**Comment by rohan09-raj** [comment]
> iPhone X view before changes: image
> 
> iPhone X view after changes: image
> 
> Samsung Galaxy fold view before changes: image
> 
> Samsung Galaxy fold view after changes: image
> 
> Note: The rest of the devices were aligned as expected.
> 
> Fix for all screens to vertically centre the text. Before changes: image
> 
> After changes: image

Thanks, look good to me. Great work🎉

File: src/components/navBar/navBar.module.scss [hunk 1]
```diff
@@ -39,12 +42,21 @@
   color: #87d870;
 }
 
-@media screen and (max-width: 500px) {
+@media screen and (max-width: 600px) {
```
Comment (harshith-venkatesh) [inline src/components/navBar/navBar.module.scss:45]: What is the reason for proceeding with `600px` here?

Comment (jjagannn) [reply src/components/navBar/navBar.module.scss:45]: Hi @harshith-venkatesh , I noticed an issue with Surface Duo earlier which was getting recognised within the 600 px range. Hence this change. I had Surface Duo in my default test devices on chrome and no specific reason.

Surface Duo view before change with 500px in range:
image

Surface Duo view after change with 600px in range:
image

Thanks.

**Review by whyDontI — CHANGES_REQUESTED** [review]

File: src/components/navBar/navBar.module.scss [hunk 2]
```diff
@@ -1,14 +1,17 @@
 .navBar {
+  display: flex;
   width: 100%;
-  background-color: #1d1283;  
+  background-color: #1d1283;
+  height: 5.25rem;
+  font: 700 1.25rem 'Roboto', sans-serif;
```
[the remark this answers is not in the document]
Comment (jjagannn) [reply src/components/navBar/navBar.module.scss:6]: @whyDontI Nikhil, i tried removing the mentioned properties from 'nabBar.module.scss' and they had an impact on the appearance. Let me know what we can retain based on below view.

No change:
image


After removal of mentioned properties from 'nabBar.module.scss':
image


image

[the remark this answers is not in the document]
Comment (whyDontI) [reply src/components/navBar/navBar.module.scss:6]: Not the best answer, you need to find the reason behind the problem and solve it.
We can't have `font` in all the elements, right?

**Review by whyDontI — CHANGES_REQUESTED** [review]
Can you please nest the scss?
Also, the `ul` seems unnecessary in the navbar, can we remove it and just have anchor tags, please?

File: src/components/navBar/navBar.module.scss [hunk 3]
```diff
@@ -1,14 +1,17 @@
 .navBar {
+  display: flex;
   width: 100%;
-  background-color: #1d1283;  
+  background-color: #1d1283;
+  height: 5.25rem;
```
Comment (whyDontI) [inline src/components/navBar/navBar.module.scss:5]: The height should not be mentioned ideally, it should adjust itself with the content.
The problem you are facing here is the underline below the anchor tag (`<a>`)  looks out of the `navbar`.
To solve this problem, you can add a `height` to your `<a>` which needs to be greater than the `underline-offset` and `decoration-thickness`.

Let me know if you don't understand the problem or the solution 😄

**Review by whyDontI — CHANGES_REQUESTED** [review]
@jjagannn Good work 👍🏽 
Can you also remove the `ul` and make the component simpler.
Now the SCSS is too much and complex.
Also nest the SCSS where possible

**Comment by jjagannn** [comment]
Navbar views after replacing ul with span tags

Web:
image

iPhoneX:
image

Samsung Galaxy S5:
image

iPad:
image

File: src/components/navBar/navBar.module.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (whyDontI) [reply src/components/navBar/navBar.module.scss:7]: Are you working on this comment?

**Comment by jjagannn** [comment]
Ah okay, I misunderstood as replacing with something else.
I’ll stick with usage of just nav and a.

Thanks,
Jagan J

**Review by whyDontI — CHANGES_REQUESTED** [review]

**Review by whyDontI — APPROVED** [review]
LGTM

**Pull request merged**